### ISSUE TITLE
Print setup setting does apply to all printers

### ISSUE THREAD

**Issue by Photon89** [shutter-project/shutter#47]
On Ubuntu 10.04 LTS with shutter 0.87.1~ppa1 from the PPA, the File->Page Setup dialog does not remember printer specific settings, it applies them to all.

Steps to reproduce:
1) File->Page Setup
2) Select a printer
3) Change Orientation to Landscape
4) Apply
Expected behaviour:
Setting applies to currently selected printer
Actual behaviour:
Setting applies to all printers

Launchpad Details: [#LP727249](https://bugs.launchpad.net/bugs/727249) Torsten Spindler - 2011-03-01 16:20:41 +0000

**Comment by Photon89** [comment]
Thanks for reporting this bug.

Launchpad Details: [#LPC](https://bugs.launchpad.net/shutter/+bug/727249/comments/1) Mario Kemper (Romario) - 2011-03-15 22:07:35 +0000

**Comment by Photon89** [comment]
I am a bit confused. I am pretty sure I understand your problem but I have tried several applications (eog, GIMP et cetera) and they all behave like Shutter. Some of them are not using the page setup dialog any longer because the settings are also available in the printing dialog but the settings are still not saved for a specific printer. 

I have created a video to demonstrate it:
http://dl.dropbox.com/u/961086/727249.ogv

Launchpad Details: [#LPC](https://bugs.launchpad.net/shutter/+bug/727249/comments/2) Mario Kemper (Romario) - 2011-06-30 20:40:20 +0000
